### PR TITLE
Update vendor directory to 1.4.1 release of origin

### PR DESCRIPTION
**Pull request by tmckayus**
Includes some changes from "gofmt -w" and update of oshinko-core in
vendor against 1.4.1 as well as the 1.4.1 vendor dir from openshift.
A few imports and calls needed to change in auth.go and cli.go

### CODE REVIEW

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -4,10 +4,10 @@ import (
 	"fmt"
 	"github.com/openshift/origin/pkg/cmd/flagtypes"
 	"github.com/openshift/origin/pkg/cmd/templates"
-	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	"github.com/spf13/cobra"
 	"io"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
```
Comment (sub-mod) [inline pkg/cmd/cli/cli.go]: are they both the same ?
cmdutil "github.com/openshift/origin/pkg/cmd/util"
cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"

**Comment by elmiko** [comment]
i tested this PR locally, all operations worked against a 1.5 openshift cluster (including scaling).

**Comment by sub-mod** [comment]
@elmiko thanks

File: pkg/cmd/cli/cli.go (hunk 1 above)
Comment (tmckayus) [inline pkg/cmd/cli/cli.go]: the RequireNoArguments() function seemed to move in the 1.4.1 release to the kube package, not sure why.

**Pull request merged**